hacks/genai-monitoring/README.md

```diff
@@ -53,14 +47,22 @@ cd movie-guru
 git checkout ghack-genkit-monitoring
 ```
 
-### Cloud setup
+### Firebase setup
```

Comment:
I moved this because I found that it was funky to not be able to copy the values in while I had them in front of me, so I think it makes more sense to clone the repo first.